clusterman/aws/auto_scaling_resource_group.py

```diff
@@ -174,16 +148,41 @@ def modify_target_capacity(
 
         autoscaling.set_desired_capacity(**kwargs)
 
-    @property
-    def min_capacity(self) -> int:
-        return self._group_config['MinSize']
+    def scale_up_options(self) -> Iterator[ClustermanResources]:
+        pass
```

Comment:
Why is this not also `NotImplementedError`?

I'm pretty sure this also fails mypy.

Reply:
It does not fail mypy, I kinda don't care whether it's `pass` or `NotImplementedError` because that's the thing I'm putting my next PR up for.